### ISSUE TITLE
And search query example support

### ISSUE THREAD

**Issue by aminahbl** [BuddhaNexus/buddhanexus#388]
At the time of writing search behaviour is unpredictable and it is very unclear what will lead to a successful search. As such the current search tips shown to the user are likely to increase frustration rather than offer support:

![Image](https://github.com/user-attachments/assets/5f49dfc1-b8d0-4c77-9b9a-ace96b951804)

As a temporary solution, the message will be simplified and the examples disabled. When search behaviour is more stable, new examples should be added to frame the user's expectations and help guide them to get the most out of what search is realistically capable of.

**Comment by sebastian-nehrdich** [comment]
Search indices are not built at the moment, so there shouldn’t be an
expectation of things working yet.

**Comment by aminahbl** [comment]
Thanks @sebastian-nehrdich! Totally makes sense.

This is just a little reminder. I was finishing up some edits on search and following our meeting on Thursday and the point you made about waiting to see how search behaves, at this point it seems to make sense to just use a more generic message rather than give specific examples. 

When things are clearer example searches can be added back again (and this ticket it just to try and help us not to forget).

**Comment by ayya-vimala** [comment]
Search seems to be doing better now. Please use Tibetan for testing.
Pali is not quite working yet. It is fixed but not deployed yet.

**Comment by aminahbl** [comment]
Yeah, this is quite low priority. For now the general tips I think are a good step. I don't have enough familiarity with any of the non-Latin scripts to make meaningful recommendations here. 

It will be a nice extra touch to give examples to the user of what will and won't work, but I think this can be put on the back burner. 

If any obvious examples do pop up during testing, please add to the ticket description.
